### ISSUE TITLE
Navigation after closing game over window

### ISSUE THREAD

**Issue by zuzaman** [YashasG98/Multiplayer-Game-Server#29]
While testing for a posted issue, I found the following:

In both games in case the windows are closed with "X", it seems it is not anymore possible to reach the main menu or to quit the game. 

I can see a navigation bar button on the right side but there are no navigation options. I was thinking of adding a way back to the main menu here. What do you think?

**Comment by YashasG98** [comment]
This seems to have been solved with #30. If you still feel some problem exists, please feel free to edit this issue, or I will close it upon your confirmation.

**Comment by zuzaman** [comment]
Yes, this was based on an incorrect interpretation of the other issue. Please go ahead to close it. Thanks.
